GitUI/BranchTreePanel/RepoObjectsTree.BranchTree.cs

```diff
@@ -39,12 +39,6 @@ protected override void OnAttached()
                 IsFiltering.Value = false;
             }
 
-            protected override Task PostRepositoryChangedAsync(GitUIEventArgs e)
-            {
-                IsFiltering.Value = false;
```

Comment:
How is `IsFiltering` used now?

Reply:
Not changed at all, it is just that the panel is updated only once.
(Maybe the property can be removed?)

Reply:
`IsFiltering` is required but `_isCurrentlyFiltering` can be removed.
However, when testing this I get a race condition causing _isCurrentlyFiltering to be set before IsFiltering so if the refresh is initiated from the grid (filter only) or Browse (could be module), the result may differ.
I changed this to only fully refresh the side panel on  actual module changes, so refreshing from grid or browse is the same. 

Cannot see a need to force the refresh.
If so this can be handled in the Loading event.

Reply:
The loaded structure must be invalidated when changed (like a branch is added) so the latest commit will need to be adjusted.